linker/Mono.Linker/Driver.cs

```diff
@@ -225,10 +225,12 @@ static IStep ResolveStep (string type)
 		static string [] ReadLines (string file)
 		{
 			ArrayList lines = new ArrayList ();
-			using (StreamReader reader = new StreamReader (file)) {
-				string line;
-				while ((line = reader.ReadLine ()) != null)
-					lines.Add (line);
+			using (FileStream fileStream = new FileStream (file, FileMode.Open)) {
+				using (StreamReader reader = new StreamReader (fileStream)) {
+					string line;
+					while ((line = reader.ReadLine ()) != null)
+						lines.Add (line);
+				}
 			}
```

Comment:
This is another one where it's not available in the [apis](https://docs.microsoft.com/en-us/dotnet/core/api/system.io.streamreader) despite existing in the source. I believe this is being reintroduced in 2.0 (see [http://stackoverflow.com/questions/40697200/no-streamreader-constructor-accepting-a-string](http://stackoverflow.com/questions/40697200/no-streamreader-constructor-accepting-a-string)), but I tried building against netcoreapp2.0 and that didn't fix the problems I was having with these APIs not existing yet - seems to be a work in progress.